src/main/java/Seat.java

```diff
@@ -0,0 +1,19 @@
+public class Seat {
+   private int seatNumber;
+   private int rowNumber;
+   private boolean isTaken;
+
+   Seat (int seatNumber, int rowNumber) {
+       this.seatNumber = seatNumber;
+       this.rowNumber = rowNumber;
+   }
+
+    void isSeatTaken (boolean IsTaken){
+       this.isTaken = IsTaken;
+   }
+
```

Comment:
brakuje mi jakiejś metody, która zwraca informację o tym, że dane miejsce jest zajęte.
aby później móc wykorzystać to w testach np:

`Assertions.assertThat(new Seat(3,5).isTaken(), true)`